src/com/pentalog/pentastagiu/homework2/shop/Basket.java

```diff
@@ -0,0 +1,52 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket {
+    private static int totalPrice;
+    private static Customer customerBakset;
+    public static List<Product> basketProductsList= new ArrayList<>();
+
+    public Basket(Customer customer) {
+        this.customerBakset = customer;
+        this.totalPrice = 0;
+    }
+
+    public  int getTotalPrice(){
+        if(customerBakset.getUserMembership() == Customer.Membership.GOLD){
+            refreshPrice();
+            totalPrice *= 0.8;
+        }
+        if (customerBakset.getUserMembership() == Customer.Membership.SILVER){
+            refreshPrice();
+            totalPrice *= 0.9;
+        }
+        if (customerBakset.getUserMembership() == Customer.Membership.NOMEMBERSHIP){
+            refreshPrice();
+            totalPrice *= 1;
+
+        }
+        return totalPrice;
+    }
+    public static void addProduct(Product product){
```

Comment:
ar trebui refacuta logica cu metodele astea statice, cum spuneam operatiile astea se intampla pe un cos, in care avem o lista de produse; alt cos va avea alta lista de produse